doc/cascadia/profiles.schema.json

```
@@ -381,7 +381,7 @@
         "background": {
           "$ref": "#/definitions/Color",
           "default": "#000000",
-          "description": "Sets the background color of the profile. Overrides background set in color scheme if colorscheme is set. Uses hex color format: \"#rrggbb\". Default \"#000000\" (black).",
+          "description": "Sets the background color of the profile. Overrides background set in color scheme if colorscheme is set. Uses hex color format: \"#rrggbb\". Default \"#0c0c0c\" (background color of Campbell).",
```

not to nitpick too much: there _is_ no default for this, and the description should not say there is. Since it's specifically an override property.. right? idk

---

Yea I'd probably just pull that last bit. 
```suggestion
          "description": "Sets the background color of the profile. Overrides background set in color scheme if colorscheme is set. Uses hex color format: \"#rrggbb\".",
```